documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -146,3 +152,6 @@ spec:
 . Wait for the Cluster Operator to update the cluster.
```

Comment:
Can we add `inter.broker.protocol.version` to the example above?

Reply:
Good spot. I added new attributes for the lower and higher inter-broker protocol versions. 

`:InterBrokVersLower: 2.6`
`:InterBrokVersHigher: 2.7`

Added the higher version to the example.